.github/PULL_REQUEST_TEMP.md

```diff
@@ -0,0 +1,59 @@
+## Related issue #
+
+<!--
+Please link the GitHub issue this pull request resolves in the format of `#1234`. If you discussed this change
+with a maintainer, please mention her/him using the `@` syntax (e.g. `@JimBugwadia`).
+
+If this change neither resolves an existing issue nor has sign-off from one of the maintainers, there is a
+chance substantial changes will be requested or that the changes will be rejected.
+
+You can discuss changes with maintainers in the [Kyverno Slack Channel](https://kubernetes.slack.com/).
+-->
+
+## Milestone of this PR
+<!--
+
+Add the milestone label by commenting `/milestone 1.2.3`.
+
+-->
+## What type of PR is this
+
+<!--
+
+> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading white spaces from that line:
+>
+> /kind api-change
+> /kind bug
+> /kind cleanup
+> /kind design
+> /kind documentation
+> /kind failing-test
+> /kind feature
+-->
+
+## Proposed Changes
+
+<!--
+Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. 
+
+***NOTE***: If this PR results in new or altered behavior which is user facing, you **MUST** read and follow the steps outlined in the [PR documentation guide](pr_documentation.md) and add Proof Manifests as defined below.
+-->
+
+## Checklist
+
+<!--
+Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of
+them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
+-->
+
+- [] I have read the [contributing guidelines](https://github.com/kyverno/kyverno/blob/main/CONTRIBUTING.md).
+- [] I have added tests that prove my fix is effective or that my feature works.
```

Comment:
Not sure if this is applicable, we typically do not add tests to the website docs. 

Perhaps we can rephrase to "I've inspected the website preview for accuracy"?